### DISCUSSION TITLE
Aspnet Core .Net8 Logging

### DISCUSSION THREAD

**Discussion by alberk8** [open-telemetry/opentelemetry-dotnet#5420] [Q&A] [answered]
I have follow this [example](https://github.com/open-telemetry/opentelemetry-dotnet/blob/main/docs/logs/getting-started-aspnetcore/Program.cs) with OtelExporter and it works when logging with _logger.LogInformation() but _logger.LogDebug() is not exported.  

What do I need to get that going?

**Comment by cijothomas** [comment, marked as answer]
https://github.com/open-telemetry/opentelemetry-dotnet/tree/main/docs/logs/customizing-the-sdk#log-filtering

Please make sure filtering is setup correctly. If you are not explicitly configuring logging filter, then the default is what gets applied. which may not be what you need.

**Reply by alberk8** [reply]
Thanks.  Is there way to define the filter in the appsettings.json file?

**Reply by cijothomas** [reply]
yes
